src/Multiselect/MultiselectComponents.jsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import classNames from 'classnames';
+import { components } from 'react-select';
+import PropTypes from 'prop-types';
+import Icon from '../Icon';
+import { Close, KeyboardArrowDown } from '../../icons';
+
+const ClearIndicator = (props) => (
+  <components.ClearIndicator {...props}>
+    <Icon src={Close} />
+  </components.ClearIndicator>
+);
+
+const DropdownIndicator = (props) => (
+  <components.DropdownIndicator {...props}>
+    <Icon src={KeyboardArrowDown} />
+  </components.DropdownIndicator>
+);
+
+const ValueContainer = ({ children, ...props }) => (
+  <components.ValueContainer {...props}>
+    {React.Children.map(children, (child) => (child && child.type !== components.Placeholder ? child : null))}
+  </components.ValueContainer>
+);
+
+const Control = ({
```

Comment:
[curious] Did you attempt to make use of `Form.Control` in any way here to avoid having to replicate CSS styles? Might not be possible or overly complex, but just wondering if we explored that avenue 😄

Reply:
I didn't try because if I place `Form.Control` instead of `Control` it will appear as `Form.Control` inside another `Form.Control` and there a lot of change will be needed vs make some style and logic updates. The only problem that took a lot of time was to set up floating label.